server.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const server = express();
+const bodyParser = require("body-parser")
+
+let mysql = require('mysql')
+
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: '',
+    database: 'users'
+})
+
+connection.connect(function(err){
+    if(err) throw err
+        console.log("You're connected.")  
+})
+
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+server.set("view-engine", "ejs")
+//server.use(express.urlencoded({ extended: false}));
+
+server.get("/", (req, res)=>{
+    res.render("index.ejs")
+})
+
+server.post("/", urlencodedParser, (req, res)=>{
+    const sql = "INSERT INTO `register` (`fname`,`lname`,`uname`,`email`,`pwd`) VALUES ('"+ req.body.fname +"', '"+ req.body.lname +"', '"+ req.body.uname +"', '"+ req.body.email +"', '"+ req.body.password +"')";
```

Comment:
too generic route path.  you should not be doing a specific task like user registration at the route path `/`